indra/newview/skins/default/xui/en/panel_region_terrain_texture_transform.xml

```diff
@@ -0,0 +1,263 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<panel
+ border="true"
+ follows="top|left"
+ height="460"
+ help_topic="panel_region_terrain_tab"
```

Comment:
this leads to a "help_browser", if there is no plan to have a help topic page, there probably houldn't be this field

Reply:
I didn't pay much attention to this field when replacing it. There are indeed no plans for a help topic.